### YamlMime:ManagedReference
items:
- uid: System.Configuration.Provider.ProviderException
  id: ProviderException
  children:
  - System.Configuration.Provider.ProviderException.#ctor
  - System.Configuration.Provider.ProviderException.#ctor(System.String)
  - System.Configuration.Provider.ProviderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Configuration.Provider.ProviderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ProviderException
  nameWithType: ProviderException
  fullName: System.Configuration.Provider.ProviderException
  type: Class
  summary: "A kivétellel, hogy amikor egy konfigurációs szolgáltató hiba fordul elő. A kivételosztály szolgáltatók által is használt a kell küldeni a kivételek az szolgáltatót, amelyet nem feleltethetők meg más már meglévő kivétel osztályok belül belső hibák esetén."
  remarks: "A ProviderException konfigurációszolgáltatójának betöltése során bármilyen hiba esetén fordul elő."
  syntax:
    content: 'public class ProviderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Configuration.Provider.ProviderException.#ctor
  id: '#ctor'
  parent: System.Configuration.Provider.ProviderException
  langs:
  - csharp
  name: ProviderException()
  nameWithType: ProviderException.ProviderException()
  fullName: System.Configuration.Provider.ProviderException.ProviderException()
  type: Constructor
  assemblies:
  - System.Configuration
  namespace: System.Configuration.Provider
  summary: "Létrehoz egy új példányt a <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> osztály."
  syntax:
    content: public ProviderException ();
    parameters: []
  overload: System.Configuration.Provider.ProviderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Configuration.Provider.ProviderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Configuration.Provider.ProviderException
  langs:
  - csharp
  name: ProviderException(String)
  nameWithType: ProviderException.ProviderException(String)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(String)
  type: Constructor
  assemblies:
  - System.Configuration
  namespace: System.Configuration.Provider
  summary: "Létrehoz egy új példányt a <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> osztály."
  syntax:
    content: public ProviderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Miért leíró üzenet ez <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> lépett fel."
  overload: System.Configuration.Provider.ProviderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Configuration.Provider.ProviderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Configuration.Provider.ProviderException
  langs:
  - csharp
  name: ProviderException(SerializationInfo,StreamingContext)
  nameWithType: ProviderException.ProviderException(SerializationInfo,StreamingContext)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Configuration
  namespace: System.Configuration.Provider
  summary: "Létrehoz egy új példányt a <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> osztály."
  syntax:
    content: protected ProviderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Az objektum, amely tartalmazza a deszerializálása."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Configuration.Provider.ProviderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Configuration.Provider.ProviderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Configuration.Provider.ProviderException
  langs:
  - csharp
  name: ProviderException(String,Exception)
  nameWithType: ProviderException.ProviderException(String,Exception)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(String,Exception)
  type: Constructor
  assemblies:
  - System.Configuration
  namespace: System.Configuration.Provider
  summary: "Létrehoz egy új példányt a <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> osztály."
  syntax:
    content: public ProviderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Miért leíró üzenet ez <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> lépett fel."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy ennek <xref href=&quot;System.Configuration.Provider.ProviderException&quot;> </xref> jelezni, hogy."
  overload: System.Configuration.Provider.ProviderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Configuration.Provider.ProviderException.#ctor
  parent: System.Configuration.Provider.ProviderException
  isExternal: false
  name: ProviderException()
  nameWithType: ProviderException.ProviderException()
  fullName: System.Configuration.Provider.ProviderException.ProviderException()
- uid: System.Configuration.Provider.ProviderException.#ctor(System.String)
  parent: System.Configuration.Provider.ProviderException
  isExternal: false
  name: ProviderException(String)
  nameWithType: ProviderException.ProviderException(String)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Configuration.Provider.ProviderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Configuration.Provider.ProviderException
  isExternal: false
  name: ProviderException(SerializationInfo,StreamingContext)
  nameWithType: ProviderException.ProviderException(SerializationInfo,StreamingContext)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Configuration.Provider.ProviderException.#ctor(System.String,System.Exception)
  parent: System.Configuration.Provider.ProviderException
  isExternal: false
  name: ProviderException(String,Exception)
  nameWithType: ProviderException.ProviderException(String,Exception)
  fullName: System.Configuration.Provider.ProviderException.ProviderException(String,Exception)
- uid: System.Configuration.Provider.ProviderException.#ctor*
  parent: System.Configuration.Provider.ProviderException
  isExternal: false
  name: ProviderException
  nameWithType: ProviderException.ProviderException
